.github/workflows/client.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
       with:
         # npm cache files are stored in `~/.npm` on Linux/macOS
         path: ~/.npm
-        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+        key: ${{ runner.os }}-build-v2-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
```

Comment:
why v2?

Reply:
This was a change that @Tengelma made. It was an issue with the cache and it is the only way to reset the cache evidently for Github actions. It is a known problem with github actions.